### PR TITLE
upd(webapp-manager): `1.2.8` -> `1.3.0`

### PR DESCRIPTION
**Pull request by Zahrun**

### CODE REVIEW

**Comment by Zahrun** [comment]
Version 1.3.0 gives this error in jammy:
```
  File "/usr/lib/webapp-manager/webapp-manager.py", line 24, in <module>
    from common import _async, idle, WebAppManager, download_favicon, ICONS_DIR, BROWSER_TYPE_FIREFOX
  File "/usr/lib/webapp-manager/common.py", line 397, in <module>
    def download_image(root_url: str, link: str) -> Optional[Image]:
  File "/usr/lib/python3.10/typing.py", line 312, in inner
    return func(*args, **kwds)
  File "/usr/lib/python3.10/typing.py", line 403, in __getitem__
    return self._getitem(self, parameters)
  File "/usr/lib/python3.10/typing.py", line 529, in Optional
    arg = _type_check(parameters, f"{self} requires a single type.")
  File "/usr/lib/python3.10/typing.py", line 176, in _type_check
    raise TypeError(f"{msg} Got {arg!r:.100}.")
TypeError: typing.Optional requires a single type. Got <module 'PIL.Image' from '/usr/lib/python3/dist-packages/PIL/Image.py'>.
```

File: packages/webapp-manager/webapp-manager.pacscript [hunk 1]
```diff
@@ -1,11 +1,15 @@
 name="webapp-manager"
-version="1.2.8"
-url="https://github.com/linuxmint/webapp-manager/archive/refs/tags/${version}.tar.gz"
+version="1.3.0"
```
Comment (Henryws) [inline packages/webapp-manager/webapp-manager.pacscript:2]: Don't have 2 versions if you can.

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/webapp-manager/webapp-manager.pacscript (hunk 1 above)
Comment (Zahrun) [reply packages/webapp-manager/webapp-manager.pacscript:2]: See my comment 

> Version 1.3.0 gives this error in jammy:
> 
> ```
>   File "/usr/lib/webapp-manager/webapp-manager.py", line 24, in <module>
>     from common import _async, idle, WebAppManager, download_favicon, ICONS_DIR, BROWSER_TYPE_FIREFOX
>   File "/usr/lib/webapp-manager/common.py", line 397, in <module>
>     def download_image(root_url: str, link: str) -> Optional[Image]:
>   File "/usr/lib/python3.10/typing.py", line 312, in inner
>     return func(*args, **kwds)
>   File "/usr/lib/python3.10/typing.py", line 403, in __getitem__
>     return self._getitem(self, parameters)
>   File "/usr/lib/python3.10/typing.py", line 529, in Optional
>     arg = _type_check(parameters, f"{self} requires a single type.")
>   File "/usr/lib/python3.10/typing.py", line 176, in _type_check
>     raise TypeError(f"{msg} Got {arg!r:.100}.")
> TypeError: typing.Optional requires a single type. Got <module 'PIL.Image' from '/usr/lib/python3/dist-packages/PIL/Image.py'>.
> ```

Comment (Henryws) [reply packages/webapp-manager/webapp-manager.pacscript:2]: So then 1.3.0 doesn't work on Jammy, so it's incompatible.

Comment (Zahrun) [reply packages/webapp-manager/webapp-manager.pacscript:2]: What if I am on jammy and I want to install webapp-manager with pacstall?

Comment (Henryws) [reply packages/webapp-manager/webapp-manager.pacscript:2]: I don't like the idea of more packages starting to use different versions in the same script.

Comment (Zahrun) [reply packages/webapp-manager/webapp-manager.pacscript:2]: Can we create scripts like webapp-manager-legacy, which would keep an older, more compatible version?

Comment (Henryws) [reply packages/webapp-manager/webapp-manager.pacscript:2]: Sure!

File: packages/webapp-manager/webapp-manager.pacscript [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 build_depends=("gettext")
 depends=("gir1.2-xapp-1.0" "python3" "python3-bs4" "python3-configobj" "python3-gi" "python3-setproctitle" "python3-tldextract" "xapps-common" "python3-pil" "libglib2.0-bin")
 description="Run websites as if they were apps"
 repology=("project: webapp-manager")
-hash="96b3c1886981e24b8704d85168f1ed7459b1a76820af031a28242e22178391c8"
+hash="8188846ea644a69b143b573dcd5d1c1472d0e968996fb3d3bb1ce6b1ead81658"
+incompatible=("*:jammy")
+url="https://github.com/linuxmint/webapp-manager/archive/refs/tags/${version}.tar.gz"
 maintainer="Zahrun <Zahrun@github.com>"
```
Comment (Henryws) [inline packages/webapp-manager/webapp-manager.pacscript:10]:
```suggestion
maintainer="Zahrun <Zahrun@github.com>"

```

**Review by Henryws — CHANGES_REQUESTED** [review]
A newline should separate the functions from the variables.

**Pull request merged**